### PR TITLE
Update ugame port for the final ugame10 version

### PR DESCRIPTION
**Pull request by deshipu**
There has been multiple uGame prototypes with differences in the pins
and other details. This updates the board definition to fit the final
uGame 10 design.

### CODE REVIEW

**Comment by deshipu** [comment]
Can I somehow re-run Travis? It seems to have stumbled on some connectivity problems.

**Comment by dhalbert** [comment]
Restarted the travis build. It's having fetch problems some times (this was not the first time).

File: ports/atmel-samd/boards/ugame10/mpconfigboard.h [hunk 1]
```diff
@@ -26,7 +26,7 @@
 #define SPI_FLASH_DIPO              3   // same as MISO pad
 
 // These are pins not to reset.
-#define MICROPY_PORT_A        (PORT_PA11 | PORT_PA16 |\
+#define MICROPY_PORT_A        (PORT_PA18 | PORT_PA16 |\
```
Comment (dhalbert) [inline ports/atmel-samd/boards/ugame10/mpconfigboard.h]: `PORT_PA18` is in `MICROPY_PORT_A` twice.

Comment (deshipu) [reply ports/atmel-samd/boards/ugame10/mpconfigboard.h]: I didn't notice that, thanks. I wonder why PA18 was there already. Oh, well, fixed now.

**Comment by deshipu** [comment]
Sorry, a change got included that shouldn't be there.

**Pull request merged**